dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/FeatureManager.java

```diff
@@ -1786,12 +1786,47 @@ boolean reportErrors(Result result, Collection<String> restrictedAccessAttempts,
             for (Chain chain : conflict.getValue()) {
                 installStatus.addConflictFeature(chain.getFeatureRequirement());
             }
+        }
 
+        if (disableAllOnConflict) {
+            // Remove all features on conflicts
+            Set<String> resolved = result.getResolvedFeatures();
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "Conflicts found in feature set, no features will be enabled:" + String.valueOf(sortedConflicts));
+            }
+            resolved.clear();
+            Tr.warning(tc, "UPDATE_DISABLED_FEATURES_ON_CONFLICT");
         }
         return reportedErrors;
 
     }
 
+    private boolean disableAllOnConflict(Result result) {
+        Map<String, Collection<Chain>> conflicts = result.getConflicts();
+        if (conflicts.isEmpty()) {
+            return false;
+        }
+        for (String featureName : result.getResolvedFeatures()) {
+            if (shouldDisableOnConflict(featureName)) {
+                return true;
+            }
+        }
+        for (Entry<String, Collection<Chain>> conflict : conflicts.entrySet()) {
```

Comment:
I think returning true whenever at least one candidate feature declares DAOC=true is not what we want here.  At this point there are no conflicts and all resolved features do not declare DAOC=true.  And from what you've (@tjwatson) advised, all of the features in the conflict chains sans the conflicting features are in the resolved set.  So... For this code to return true we can surmise that all resolved features declare DAOC=false or lack the DAOC header, there may multiple candidates per conflict feature, no conflict feature is a root feature, and at least one conflict feature candidate has DAOC=true.  Also, that the feature that declares the tolerated conflict feature with DAOC=true is new and does not have header=true (e.g. new auto feature or new malformed feature.)  If the candidate was introduced by a new auto feature, we should not clear the resovled set.  So, I think this method should return true iff all candidates declare DAOC=true, and we handle future issues involving new malformed features as they come.

Reply:
Fixed with latest